src/ui/simulator/toolbox/input/renewable-cluster.cpp

```diff
@@ -404,11 +385,8 @@ void RenewableCluster::internalAddPlant(void*)
             sFl.clear() << "new cluster " << indx;
         }
 
-        // gp : to be adapted
-        /*
         // We have to rebuild the scenario builder data, if required
         ScenarioBuilderUpdater updaterSB(*study);
-        */
```

Comment:
Why uncomment these lines in this PR ? This PR is about deleting clusters, not creating new ones, right ?